specifyweb/frontend/js_src/lib/components/QueryBuilder/Results.tsx

```diff
@@ -312,55 +322,58 @@ export function QueryResults(props: Props): JSX.Element {
               )}
             </div>
           </div>
+        ) : null}
+        {visibleFieldSpecs.length > 0 && (
+          <div role="rowgroup">
+            {showResults &&
```

Comment:
can you move the `visibleFieldSpecs.length > 0` condition here (`{showResults && ...`)?

this way, you can remove one extra level of indent

or even better, just add `visibleFieldSpecs.length > 0` to the definition of `showResults` and see if that works